package.json

```diff
@@ -28,6 +28,7 @@
     "lint:fix": "npm run lint -- --fix",
     "formatlib": "find warplib/ -iname *.cairo -exec cairo-format -i {} +",
     "test": "yarn warplib && tests/behaviour/setup.sh && npx mocha \"tests/**/*.test.ts\" --extension ts --require ts-node/register --parallel --exit",
+    "test:forwarder": "yarn tsc && npx mocha tests/interface_call_forwarder/interface_forwarder.test.ts --extension ts --require ts-node/register --exit",
```

Comment:
let's also make sure this test run in CI

Reply:
CI already covers it